bin/modules/beer_agent/commands.dart

```diff
@@ -0,0 +1,176 @@
+    ..appendBold(BeerAgentModule().beerSales.length.toString())
+    ..append(' current releases!')
+    ..appendNewLine()
+    ..appendNewLine();
+
+  for (var beerSale in BeerAgentModule().beerSales) {
+    var saleDate = beerSale.saleDate;
+    var saleSize = beerSale.beerList.length;
+    beerSale.beerList.shuffle();
+
+    if (saleSize >= 3) {
+      oelMessage
+        ..append(':beer: ')
+        ..appendBold(saleDate)
+        ..appendNewLine()
+        ..append('This release has ')
+        ..appendBold(saleSize)
+        ..append(' new beers!')
+        ..appendNewLine()
+        ..appendNewLine()
+        ..append('Some of them are:')
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[0].name)
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[1].name)
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[2].name)
+        ..appendNewLine()
+        ..appendNewLine();
+    } else if (saleSize == 2) {
+      oelMessage
+        ..append(':beer: ')
+        ..appendBold(saleDate)
+        ..appendNewLine()
+        ..append('This release has ')
+        ..appendBold(saleSize)
+        ..append(' new beers!')
+        ..appendNewLine()
+        ..appendNewLine()
+        ..append('Some of them are:')
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[0].name)
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[1].name)
+        ..appendNewLine()
+        ..appendNewLine();
+    } else if (saleSize == 1) {
+      oelMessage
+        ..append(':beer: ')
+        ..appendBold(saleDate)
+        ..appendNewLine()
+        ..append('This release has ')
+        ..appendBold(saleSize)
+        ..append(' new beer!')
+        ..appendNewLine()
+        ..appendNewLine()
+        ..append('- ')
+        ..appendBold(beerSale.beerList[0].name)
+        ..appendNewLine()
+        ..appendNewLine();
+    }
+  }
+
+  oelMessage
+    ..append('---')
+    ..appendNewLine()
+    ..append('For more information: https://systembevakningsagenten.se/')
+    ..appendNewLine()
+    ..appendNewLine()
+    ..append('Cheers! :beers:');
+
+  //Send message
+  await ctx.respond(oelMessage);
+}
+
+Future<void> _releaseCommand(ISlashCommandInteractionEvent ctx) async {
+  var input = ctx.args;
+  if (input.length == 1) {
+    var parsedDate = DateTime.tryParse(input[0].value);
+
+    if (parsedDate != null) {
+      await BeerAgentModule()._updateBeerSales();
+      for (var sale in BeerAgentModule().beerSales) {
+        var saleDate = DateTime.parse(sale.saleDate);
+        if (parsedDate == saleDate) {
+          //Compile beer list to string and sort by name.
+          var beerStr = '';
+          sale.beerList.sort((a, b) => a.name.compareTo(b.name));
+          sale.beerList.forEach((element) {
+            beerStr += '- ' + element.name + '\n';
+          });
```

Comment:
I think this could be accomplished with join?
Instead of foreaching with a saved variable, we could join with `'- ' + element.name + '\n'`

https://api.flutter.dev/flutter/dart-core/Iterable/join.html